Library/Homebrew/diagnostic.rb

```diff
@@ -715,6 +715,9 @@ def check_for_bad_python_symlink
         return if Regexp.last_match(1).nil?
         return if Regexp.last_match(1) == "2"
 
+        # Disable this check on Linux because Ubuntu 20.04's python executable is for Python 3
+        return if OS.linux?
```

Comment:
It looks like we could move this condition to the first line in the method — there's no reason to call `which` or `python -V` on Linux.

Reply:
Oh yeah, good point. 👍